lib/cs_guide/postcode_lat_long.ex

```diff
@@ -24,29 +23,82 @@ defmodule CsGuide.PostcodeLatLong do
         # against the cache. For testing at that point we can create a smaller
         # csv file which we can use to test with.
       _ ->
-        postcode_cache = :ets.new(:postcode_cache, [:set, :protected, :named_table])
-        store_postcodes_in_ets(postcode_cache)
+        :ets.new(:postcode_cache, [:set, :protected, :named_table])
```

Comment:
was variable assignment spiking the memory usage? 🤔 (_good catch!_)

Reply:
😂 every bit counts. (I actually just removed it because it didn't really need to be assigned)